pages/emotions/[slug].js

```diff
@@ -0,0 +1,42 @@
+import { emotionData } from "@/lib/data";
+import { useRouter } from "next/router";
+import EmotionDetails from "@/components/EmotionDetail";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 8.5px;
+  &:hover {
+    background-color: ${({ $color }) => $color};
+  }
+`;
+
+export default function EmotionDetailsPage() {
+  console.log(emotionData);
```

Comment:
I think we should remove the console logs before merging into main (although i am a big fan of logging during development). 
```suggestion
```